src/NUnitFramework/testdata/TestCaseOrderAttributeFixture.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework;
+
+namespace NUnit.TestData
+{
+    [TestFixture]
+    public class TestCaseOrderAttributeFixture
+    {
+        [Test]
+        [Order(3)]
+        public void Z_ThirdTestWithSameOrderAsSecond()
```

Comment:
Originally I had the sort by name as a secondary sort, so was testing that, but as Charlie mentioned before he requested it removed, I've updated the wording on the method now to just Third Test.